include/ttmlir/Dialect/TTIR/Transforms/Passes.td

```diff
@@ -125,4 +125,12 @@ def TTIRBroadcastFold: Pass<"ttir-broadcast-fold", "::mlir::ModuleOp"> {
   }];
 }
 
+def TTIRHoistTransform: Pass<"ttir-cpu-hoist-transform", "::mlir::ModuleOp">
+{
+  let summary = "Transform to perform hoist mechanics on any ops marked to be hoisted for CPU lowering";
+  let description = [{
+    ..
```

Comment:
We should use this description to outline the flow.  i.e. divided into stages, first analysis passes run to determine which ops to hoist followed by a mechanical pass that does the hoisting + whatever other high level details it does to the IR to make this happen.